### PR TITLE
Rename yii-queue -> queue, Yii Queue -> Queue, Yii\Queue -> Queue

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ✔️

### CODE REVIEW

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
https://github.com/yiisoft/yii-queue-amqp should be renamed as well.

**Review by s1lver — APPROVED** [review]

**Review by rustamwin — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Yii Queue Change Log
+# Queue Change Log
```
Comment (vjik) [inline CHANGELOG.md:1]:
```suggestion
# Yii Queue Change Log
```

We change package name to `yiisoft/queue` only, but pretty name keep as `Yii Queue`

File: Makefile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-export COMPOSE_PROJECT_NAME=yii-queue
+export COMPOSE_PROJECT_NAME=queue
```
Comment (vjik) [inline Makefile:1]:
```suggestion
export COMPOSE_PROJECT_NAME=yii-queue
```

This name is used in globlal namespace. I think, better keep `yii-` prefix.

File: tests/docker/docker-compose.yml [hunk 3]
```diff
@@ -1,8 +1,8 @@
 version: '3.8'
 
 services:
-  yii-queue-php:
-    container_name: yii-queue-php
+  queue-php:
+    container_name: queue-php
```
Comment (vjik) [inline tests/docker/docker-compose.yml:5]:
```suggestion
    container_name: yii-queue-php
```

It's used in global namespace. Better to keep `yii-` prefix.

**Review by vjik — CHANGES_REQUESTED** [review]

**Pull request merged**